src/components/numeral-date/numeral-date.stories.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { useState } from "react";
+import { ComponentStory } from "@storybook/react";
+import NumeralDate from ".";
+
+export const Default: ComponentStory<typeof NumeralDate> = () => {
```

Comment:
If we're going to rename this to `Default` it should not be the controlled implementation, it should be the minimum props required for the component to work. When we remove the support for uncontrolled this will become the default

Reply:
ok, story name changed